### PR TITLE
Add setup ledger info

### PR DESCRIPTION
**Pull request by badrogger**

### CODE REVIEW

File: validators/register-validator-node.md [hunk 1]
```diff
@@ -499,11 +517,11 @@ Optional arguments:
 sk-val validator register -n SETeam -d "SE Team description" -c 20 --min-delegation 0 --pk-file ./pk.txt
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md]: If it's ledger wallet they do not need to provide --pk-file

File: validators/register-validator-node.md [hunk 2]
```diff
@@ -469,7 +469,25 @@ Usage example:
 sk-val init -e [ENDPOINT] -c [ABI] --wallet software
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md]: It's better for them to provide [software/hardware]

**Review by ebruengwall-ant — COMMENTED** [review]
It would be great if we separate software and hardware wallet setup and make it clear information

**Review by christineRPM — DISMISSED** [review]

**Review by gannakulikova — DISMISSED** [review]

**Review by christineRPM — DISMISSED** [review]

File: validators/register-validator-node.md [hunk 3]
```diff
@@ -550,65 +575,106 @@ Signature: <your-signature>
...
 ```
 
+Required arguments:
+
+-   `--delegation-id` - Delegation id to accept
+
+Optional arguments:
+
+-   `--pk-file` - Path to file with private key (only for software wallet type)
+-   `--gas-price` - Gas price value in Gwei for transaction (if not specified doubled average network value will be used)
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md:657]: I wish the average was 1.25 default not 2 I'll approve but can we create a ticket for this?

File: validators/register-validator-node.md [hunk 4]
```diff
@@ -550,65 +575,106 @@ Signature: <your-signature>
...
  sk-val holder delegate --validator-id [Validator_ID] --amount 100 --delegation-period 3 --pk-file pk.txt --info "please accept delegation" --yes
 ```
+Required arguments:
+
+-   `--validator-id` - Validator id
+
+-   `--amount` - Delegation amount
+info)
+
+-   `--delegation-perion` - Delegation period (only 2 month allowed for
```
Comment (ebruengwall-ant) [inline validators/register-validator-node.md]: delegation-period typo

**Review by christineRPM — APPROVED** [review]

**Review by ebruengwall-ant — APPROVED** [review]

**Pull request merged**